modules/payments/client-react/stripe/subscription/containers/AddSubscription.tsx

```diff
@@ -24,8 +24,7 @@ class AddSubscription extends React.Component<AddSubscriptionProps, { [key: stri
   constructor(props: AddSubscriptionProps) {
     super(props);
     this.state = {
-      submitting: false,
-      error: null
```

Comment:
Looks like you forgot to remove the use of "error" field from the render function

Reply:
Fixed